content/rule_book/quiz_meets.md

```diff
@@ -0,0 +1,172 @@
+# Quiz Meets
+
+The tournament brackets are based on 3 things: A preliminary round (or "prelims"), an elimination round (or "brackets") at the discretion of the meet director, and championship quizzes.
+
+## Preliminary Rounds
+
+Team points are calculated using a team's score at the end of question 20, together with their place, which could be determined at the end of question 20 or after the end of overtime. Teams start with a base number of points based on their place and receive additional points based on their team score. The specifics are detailed in the section below.
+
+In case of a tie, points are awarded according to the team score at the end of question 20. Overtime is used solely to determine placements.
+
+If ties are not being broken in prelims, more than 1 team can receive 1st place, 2nd place, or 3rd place points.
+
+### Team Points Calculation
+
+Team points are calculated by dividing the team score immediately after question 20 by 10 and then applying placement adjustment:
+
+- No change for 1st
```

Comment:
Jessica Tower of WGLD wrote regarding "6.1.1. Team Points Calculation":

> If I am reading this correctly, this is combining the two ways of computing
> team score into one, which results in a different score. There are two ways I
> was taught to compute score:
>
> 1. Dividing team points by 10, then
> no change for 1st place,
> -1 for 2nd place, and
> -2 for 3rd place (keeping in mind the minimum points awarded are 10, 5, 1)
>
> OR
>
> 2. 1st place gets 10 points plus one point for each 10 points over 100; 2nd
> place gets 5 points plus 1 point for each 10 points over 60; 3rd place gets 1
> point plus 1 point for each 10 points over 30.
>
>
> These two scoring systems end up with teams being awarded the same number of
> points, but if you combine them (as it seems above), then that is a NEW
> scoring system that will end up giving different points.
>
> Example with teams scoring 170, 150, 130 in a quiz:
>
> Both #1 and #2 scoring systems will result in team points of 17, 14, and 11.
> But if you use the scoring system in 6.1.1 the team points would be 18, 15,
> and 12.
>
> Personally I use #1 scoring system because it is less for my brain to keep
> track of to just divide by 10 and subtract 1 or 2 points. I think #2 is
> confusing.
>
> 6.1.1 looks like it is trying to approximate the ease of #1, except that the
> +1, 0, -1 are different.

Zachary Tinker replied:

> By changing this scoring it did not appear to me to create that big of a
> change. All scores are dropped by 1 so then it equal across the board
> resulting in the same outcome, just with slightly different numbers. However
> there is one effect it does create. It creates a bigger gap between all of the
> placements if the others are at their minimums.
>
> Example:
>
> 1st = 180 | 2nd = 50 | 3rd = 20
> 2018 RB | 18 > 5 > 1
> 2021 RB | 19 > 5 > 1
>
> This creates a larger gap than has previously existed for the other two teams
> to overcome to compensate. Is that bad? I'm not sure. But is does change
> things and quite potentially the strategy used by coaches.
>
> If we are aiming for complete FE, then I say patch it. Since scoring in 2018
> was not a bug.

[This patch](c9c131d) fixes this change (which is beyond our functional equivalence
mandate) in this initial version of the rule book.